edgedb/asyncio_con.py

```diff
@@ -71,6 +71,31 @@ async def _fetchall(
         __limit__: int=0,
         __typeids__: bool=False,
         __typenames__: bool=False,
+        __allow_capabilities__: typing.Optional[int]=None,
+        **kwargs,
+    ) -> datatypes.Set:
+        result, _ = await self._protocol.execute_anonymous(
+            query=query,
+            args=args,
+            kwargs=kwargs,
+            reg=self._codecs_registry,
+            qc=self._query_cache,
+            implicit_limit=__limit__,
+            inline_typeids=__typeids__,
+            inline_typenames=__typenames__,
+            io_format=protocol.IoFormat.BINARY,
+            allow_capabilities=__allow_capabilities__,
+        )
+        return result
+
+    async def _fetchall_with_headers(
+        self,
+        query: str,
+        *args,
+        __limit__: int=0,
+        __typeids__: bool=False,
+        __typenames__: bool=False,
+        __allow_capabilities__: typing.Optional[int]=None,
         **kwargs,
     ) -> datatypes.Set:
```

Comment:
Change the type annotation to `typing.Tuple[datatypes.Set, ???]`, replacing ??? with the correct type of returned headers

Reply:
Fixed.